icons/yii/yii-plain-wordmark.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M124.463 39.561c1.99 0 3.537 1.548 3.537 3.538 0 1.99-1.547 3.538-3.537 3.538s-3.538-1.548-3.538-3.538c0-1.77 1.547-3.538 3.538-3.538Zm-12.824 0c1.989 0 3.538 1.548 3.538 3.538 0 1.99-1.548 3.538-3.538 3.538-1.989 0-3.538-1.548-3.538-3.538 0-1.77 1.549-3.538 3.538-3.538Zm3.538 43.334V49.953c0-.663-.443-1.106-1.106-1.106h-4.864c-.663 0-1.106.443-1.106 1.106v32.942c0 .663.443 1.106 1.106 1.106h4.864c.663-.222 1.106-.663 1.106-1.106Zm-12.603.884V49.07c-2.653 0-6.854 1.768-6.854 5.527v23.656c-1.547.222-3.759.443-5.084.443-4.423 0-5.527-3.096-5.527-7.297V54.374c-.222-4.2-4.865-5.306-6.854-5.306v22.11c0 7.958 3.98 13.043 12.38 13.043 1.326 0 3.98-.443 5.085-.663l.058.573c.306 6.887-7.354 4.07-11.775 3.628-3.317 0-5.527 2.653-6.192 5.748 1.99.663 8.182 1.327 12.603 1.769 7.96.221 12.16-4.2 12.16-11.497Zm25.205-.884V49.953c0-.663-.442-1.106-1.105-1.106h-4.864c-.664 0-1.106.443-1.106 1.106v32.942c0 .663.442 1.106 1.106 1.106h4.864c.663-.222 1.105-.663 1.105-1.106ZM46.452 74.07c-.52-6.875-2.739-11.332-3.819-13.859-1.08-2.525-2.739-4.89-2.74-4.886-.003.011-1.495 8.047-4.175 13.86-.448.971-1.042 2.184-1.642 3.16-1.851 3.268-4.537 6.395-6.137 9.563-1.588 3.141-1.882 6.251-1.735 9.787.15 3.556.97 7.044 1.76 10.501 2.973-.642 5.56-1.739 7.799-3.142 5.89-3.694 9.45-9.6 10.462-15.965 0 0 .05-.258.071-.573.456-4.919.305-6.47.156-8.445Z"/><path d="M46.452 74.07c-.52-6.875-2.739-11.332-3.819-13.859-1.08-2.525-2.739-4.89-2.74-4.886v.002-.002l-.396-.59C30.809 42.542 14.149 36.453.072 42.096c-.678 8.549 3.285 23.256 17.743 27.358 5.845 1.798 10.525 1.332 16.261 2.892 0 0 5.83 2.032 9.223 5.08 1.525 1.37 3.053 3.173 2.977 5.318.483-5.105.328-6.666.176-8.672Z"/><path fill-rule="evenodd" d="M40.139 48.169c-2.055-6.01-1.184-10.037 2.574-15.641 1.793-2.674 4.887-5.776 7.6-7.528 10.943 6.858 14.99 19.63 10.965 31.493-2.929 8.632-5.677 12.251-12.625 21.132.81-9.511-2.538-16.105-5.825-23.432-.836-1.864-1.993-3.986-2.69-6.024Z" clip-rule="evenodd"/><path d="M46.276 82.743c.076-2.145-1.452-3.948-2.978-5.319-3.392-3.047-9.222-5.079-9.222-5.079.6-.976 1.194-2.189 1.642-3.16 2.68-5.813 4.172-13.849 4.175-13.86.001-.003 1.66 2.36 2.74 4.886 1.08 2.527 3.298 6.983 3.82 13.86.15 2.006.306 3.567-.178 8.672Z"/></svg>
```

Comment:
Paths need to be united

Reply:
Fixed all icons. Also checked on icomoon to make sure they convert correctly to font.